src/librustc_incremental/persist/dirty_clean.rs

```diff
@@ -53,11 +58,144 @@ use rustc_data_structures::fx::{FxHashSet, FxHashMap};
 use syntax_pos::Span;
 use rustc::ty::TyCtxt;
 
-const LABEL: &'static str = "label";
-const CFG: &'static str = "cfg";
+const EXCEPT: &str = "except";
+const LABEL: &str = "label";
+const CFG: &str = "cfg";
+
+// Base and Extra labels to build up the labels
+
+/// DefNodes for Hir, which is pretty much everything
```

Comment:
It's `DepNode`, not `DefNode`. It is, however, `DefPath`, which can be rather confusing. "Dep" is for "dependency", "Def" is for "definition".